alembic/lsstcam/versions/9fb74a42697a_update_the_views.py

```diff
@@ -0,0 +1,180 @@
+"""Update the views
+
+Revision ID: 9fb74a42697a
+Revises: e65e9e2237ba
+Create Date: 2025-04-17 16:33:24.365593+00:00
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision: str = "9fb74a42697a"
+down_revision: Union[str, None] = "e65e9e2237ba"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint(
```

Comment:
I'm surprised that this constraint wasn't already in a migration.  But I agree it's needed.